### DISCUSSION TITLE
How to exactly unbind key mapping

### DISCUSSION THREAD

**Discussion by urayoru113** [nvim-tree/nvim-tree.lua#1647] [Q&A]
In my .vimrc and lua setting

I've mapped `nnoremap <tab> gt` in .vimrc  , then I configure lua setting as below
```
mappings = {
  list = {
      { key = "<tab>", action = "" }
  },
}
```
When the cursor in tab `NvimTree_{i}`, key `tab` will be hijacked, so I couldn't press `tab` to go to the next tab page.

How can I solve this problem?

**Comment by alex-courtis** [comment]
`"<tab"` is missing the closing `>`

**Reply by urayoru113** [reply]
> `"<tab"` is missing the closing `>`

Sorry, I've edit it.
This is not the reason why `<tab>` key no function

**Reply by alex-courtis** [reply]
You will need to use a capital T. See my config that also unbinds tab: [nvim-tree.lua](https://github.com/alex-courtis/arch/blob/525e4cc19efdf2a76b7dba45ccf9f67a030c0c10/config/nvim/lua/amc/nvim-tree.lua)

This will be fixed via #1579 however I might put a quick fix in now. Thanks for letting me know!

**Reply by urayoru113** [reply]
Thanks. That works for me.

I didn't notice that because when I binded `<tab>` to other function such as `{ key = "<tab>", action = "tabnew" }` in **lowercase** `t`, it will work with no problem.

**Reply by alex-courtis** [reply]
Thank you. Raised #1676
